test/archive.cpp

```diff
@@ -377,14 +377,16 @@ struct ZimFileInfo {
 struct TestDataInfo {
   const char* const name;
   zim::entry_index_type mediaCount;
-  ZimFileInfo withnsInfo, nonsInfo;
+  ZimFileInfo withnsInfo, nonsInfo, nov0Info;
```

Comment:
`nov0` will stop making much sense after this PR is merged. `noTitleListingV0` has better chances to survive the upcoming nuclear winter. Note that the commit message will have to updated too.